okreplay-sample/build.gradle

```diff
@@ -16,48 +22,67 @@ android {
 
   defaultConfig {
     applicationId "okreplay.sample"
+    compileSdk 31
     minSdkVersion 21 // so we get faster multidex
     targetSdkVersion androidConfig.targetSdkVersion
     testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
-    vectorDrawables.useSupportLibrary = true
-    multiDexEnabled true
   }
 
   lintOptions {
     disable 'InvalidPackage'
   }
 
+  compileOptions {
+      sourceCompatibility JavaVersion.VERSION_1_8
+      targetCompatibility JavaVersion.VERSION_1_8
+  }
+
+  kotlinOptions {
+      jvmTarget = '1.8'
+  }
+
   useLibrary 'android.test.runner'
   useLibrary 'android.test.base'
   useLibrary 'android.test.mock'
 }
 
+configurations.all {
+  resolutionStrategy {
+    force "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$versions.kotlinVersion"
+    force "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$versions.kotlinVersion"
+  }
+}
+
 dependencies {
-  debugImplementation project(':okreplay-core')
-  releaseImplementation project(':okreplay-noop')
+  // debugImplementation project(':okreplay-core') // when you're working on the library
+  debugImplementation "com.airbnb.okreplay:okreplay:latest.release"
+  // releaseImplementation project(':okreplay-noop') // when you're working on the library
+  releaseImplementation "com.airbnb.okreplay:noop:latest.release"
   implementation dep.appcompat
   implementation dep.androidXAnnotations
   implementation dep.kotlinStdLib
   implementation dep.guava
-  implementation "com.google.android.material:material:1.0.0"
-  implementation 'androidx.multidex:multidex:2.0.1'
-  implementation "androidx.vectordrawable:vectordrawable:1.0.1"
-  implementation 'com.squareup.retrofit2:converter-moshi:2.3.0'
-  implementation 'com.squareup.retrofit2:adapter-rxjava2:2.3.0'
-  implementation 'com.squareup.retrofit2:converter-moshi:2.3.0'
+  implementation "com.google.android.material:material:1.5.0"
```

Comment:
Is this just demo-related UI? 

Material sometimes does breaking design changes in minor releases, so we'd have to visit the page to make sure it still looks right

Reply:
Yeah, this is just a demo app, and it does have a couple UI tests.